### PR TITLE
Edit/clean up

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

No

### Please describe the goal of this pull request and the changes that were made.

Needs further refinement, but due to time sensitiveness, I thought this might be a good starting point for website revisions.

### CODE REVIEW

File: content/about/open-source.md [hunk 1]
```diff
@@ -5,18 +5,13 @@ type: essay
...
-
-As Quire continues to evolve, our goal is to launch as fully open source by 2022 and expand Quire’s reach beyond our dedicated group of beta users. A community-supported model would enable Quire to remain free to use and ensure a long lifespan beyond Getty.
-
-<div class="action-button">
-
-[Sign Up to Use Quire Today!](https://docs.google.com/forms/d/e/1FAIpQLScKOJEq9ivhwizmdazjuhxBII-s-5SUsnerWmyF8VteeeRBhA/viewform)
-</div>
+Quire users have full access and can create, customize, and distribute their digital projects any way they'd like. We encourage users to share questions, frustrations, and positive reactions, and we use this feedback to inform developments and new feature considerations. In return, our users not only have access to Getty staff for assistance, but they also have access to other beta users who can provide additional support and inspiration. The one thing we ask is that users do NOT redistribute Quire itself.
```
Comment (geealbers) [inline content/about/open-source.md:14]: We need to remove the mention of beta, and that we don't allow redistribution. Like:

```suggestion
Quire users have full access and can create, customize, and distribute their digital projects any way they'd like. We encourage users to share questions, frustrations, and positive reactions in our [community forum](https://github.com/thegetty/quire/discussions), and we use this feedback to inform developments and new feature considerations.
```

File: content/home.md [hunk 2]
```diff
@@ -27,23 +27,10 @@ Online ∙ Print ∙ E‑book
...
-
-[Sign-Up for Quire](https://docs.google.com/forms/d/e/1FAIpQLScKOJEq9ivhwizmdazjuhxBII-s-5SUsnerWmyF8VteeeRBhA/viewform)
-
-</div>
-
-<div class="sub-action-button">
-
-[View on GitHub ![logo](/img/icons/github.svg) ](https://github.com/thegetty/quire/)
-
-</div>
```
Comment (geealbers) [inline content/home.md:44]: What do you think about leaving these in but having the button be "Get Started"? It's duplicative of the item in the top menu, but I think it's a natural place for people to look and to find a call to action. I also think there's value in still having the GitHub link up there.

**Review by geealbers — COMMENTED** [review]
Looks good. Just had a couple suggestions/queries

**Comment by Erin-Cecele** [comment]
@geealbers ok, I changed the top button to Get Started but then renamed the link to the tutorial at the bottom to "Try Quire". Not sure if that works. What do you think?

**Comment by geealbers** [comment]
"Try Quire" is ok, but why the change at all?

**Comment by Erin-Cecele** [comment]
Originally it was linked to signing up for Quire. I feel like it's useful to get the tutorial somewhere on the homepage because its very helpful for the non-technical people out there, and I like the accessible messaging around "give it a try". But like I said earlier, maybe once we get the trailer up, we can accompany it with the tutorial. That makes sense. I'll change it to a third occurrence of Get Started for now.

**Pull request merged**